### PR TITLE
tools: scripts: *.mk: Naming & typo corrections

### PR DESCRIPTION
**Pull request by danmois**
General solution to ignored error when attempting to delete non-existent file/directory in make. 

Modifications for consistency in names used in *.mk files.

### CODE REVIEW

File: tools/scripts/generic.mk [hunk 1]
```diff
@@ -24,8 +24,12 @@ ROOT_DRIVE = C:
 TIMESTAMP = 00:00:00
 copy_fun = xcopy /F /Y /B "$(subst /,\,$1)" "$(subst /,\,$2)"
 copy_folder = xcopy /F /S /Y /C /I "$(subst /,\,$1)" "$(subst /,\,$2)"
-remove_fun = del /S /Q $(subst /,\,$1)
-remove_dir = rd /S /Q $(addsuffix ",$(addprefix ",$(subst /,\,$1)))
+remove_fun_action = del /S /Q $(subst /,\,$1)
+remove_fun = $(if $(wildcard $1),$(call remove_fun_action,$(wildcard $1)),\
+			@echo rd /S /Q $(addsuffix ",$(addprefix ",$(subst /,\,$1))))
```
Comment (adrimbarean) [inline tools/scripts/generic.mk:29]: Should it be `del` instead of `rd` here?

**Review by adrimbarean — APPROVED** [review]
Looks good to me. Just a question and a comment.

File: tools/scripts/generic.mk (hunk 1 above)
Comment (danmois) [reply tools/scripts/generic.mk:29]: This is because on Windows files and folders are deleted differently: del is for files and rd is for directories.

**Comment by adrimbarean** [comment]
Commit title of the second commit is too long and shown oddly in second line.

**Comment by danmois** [comment]
V1

- changed the name of the second commit to a shorter text

**Review by amiclaus — APPROVED** [review]

**Review by dbogdan — APPROVED** [review]

**Pull request merged**